.gitignore

```diff
@@ -29,6 +29,9 @@ pip-log.txt
 .coverage
 .tox
 nosetests.xml
+coverage.xml
+test-reports
+.pytest_cache
```

Comment:
Directories should have a trailing forward slash.

Reply:
👍